### DISCUSSION TITLE
Is there a way to limit deployments by vercel bot?

### DISCUSSION THREAD

**Discussion by GabenGar** [vercel/community#485] [Help] [answered]
I have a custom domain configured on vercel. I want vercel bot only do deployments to this domain, without creating a deploy for each branch and commits within them on vercel itself. How do you do that?

**Comment by samcx** [comment, marked as answer]
You can use Ignored Build Step to cancel Deployments from other branches → https://vercel.com/support/articles/how-do-i-use-the-ignored-build-step-field-on-vercel.
